build.sbt

```diff
@@ -48,22 +46,20 @@ lazy val scaladget = project.in(file("scaladget")) settings(
   libraryDependencies += "com.lihaoyi" %%% "scalatags" % scalatagsVersion,
   libraryDependencies += "com.lihaoyi" %%% "scalarx" % rxVersion,
   npmDependencies in Compile += "bootstrap.native" -> bootstrapNativeVersion
-  ) enablePlugins(ScalaJSBundlerPlugin, ScalaJSWeb) settings (defaultSettings: _*)
+  ) enablePlugins(ScalaJSBundlerPlugin) settings (defaultSettings: _*)
 
 
 lazy val runDemo = taskKey[Unit]("runDemo")
 
-lazy val demo = project.in(file("demo")) dependsOn (scaladget) enablePlugins(ScalaJSBundlerPlugin, WebScalaJSBundlerPlugin) settings (defaultSettings: _*) settings(
+lazy val demo = project.in(file("demo")) dependsOn (scaladget) enablePlugins(ScalaJSBundlerPlugin) settings (defaultSettings: _*) settings(
   libraryDependencies += "com.lihaoyi" %%% "scalarx" % rxVersion,
   libraryDependencies += "com.lihaoyi" %%% "sourcecode" % sourceCodeVersion,
   libraryDependencies += "org.scala-lang" % "scala-reflect" % ScalaVersion % "provided",
-  scalaJSModuleKind := CommonJSModule,
   runDemo := {
     val demoTarget = target.value
     val demoResource = (resourceDirectory in Compile).value
-    IO.copyFile((fullOptJS in Compile).value.data, demoTarget / "js/demo.js")
```

Comment:
The output of `fullOptJS` is not executable anymore by the web browser (since it is a CommonJS module), so you must not include it to your assets.